### PR TITLE
Remove inline css from markup

### PR DESCRIPTION
**Pull request by willbarrett**
Includes a commit from https://github.com/danschultzer/premailex/pull/13, so this PR will get smaller once that one is merged.

Inspired by code in https://github.com/danschultzer/premailex/pull/9/files, updated to support multiple parsers.

@danschultzer

### CODE REVIEW

File: lib/premailex/html_parser/meeseeks.ex [hunk 1]
```diff
@@ -6,7 +6,8 @@ defmodule Premailex.HTMLParser.Meeseeks do
   require Logger
   import Meeseeks.CSS
   alias Premailex.HTMLParser
-  alias Meeseeks.{Selector.CSS.Parser.ParseError.Document}
+  alias Meeseeks.{Selector.CSS.Parser.ParseError}
+  alias Meeseeks.Document
```
Comment (mertonium) [inline lib/premailex/html_parser/meeseeks.ex]: If you want, you can leave `Document` in the previous line, just separate it from the other module with a comma. I.e.
```elixir
alias Meeseeks.{Selector.CSS.Parser.ParseError, Document}
```
although I think it's more obvious the way you changed it.

Comment (danschultzer) [reply lib/premailex/html_parser/meeseeks.ex]: Yeah, it's fine with the two aliases. You can remove the curly brackets, I forgot to remove them as I was working on the Meeseeks module.

**Comment by danschultzer** [comment]
Looks good @willbarrett, and sure no problem. Thanks for your help improving the library!

I've some thoughts on this PR but got some unrelated work to take care of beforehand. I'll get back to it ASAP.

**Comment by danschultzer** [comment]
I'll close this in favor of #18

**Pull request closed without merging**